talisman.go

```diff
@@ -56,6 +58,8 @@ func main() {
 	flag.BoolVar(&scan, "scan", false, "scanner scans the git commit history for potential secrets")
 	flag.StringVar(&checksum, "c", "", "short form of checksum calculator")
 	flag.StringVar(&checksum, "checksum", "", "checksum calculator calculates checksum and suggests .talsimarc format")
+	flag.StringVar(&reportdirectory, "reportdirectory", "", "directory where the scan reports will be stored")
```

Comment:
Set default value here instead of passing blank string around.

Reply:
Setting a default value here would again require an if-else check in report.go (line 21), that has been removed after the review comments.

Reply:
Capturing our offline discussion:
Irrespective of the value "reportdirectory" configuration, a sub-directory with the name talisman_reports will be created.
If reportdirectory is empty, then talisman_reports sub-directory will be created in talisman's invocation directory, if not it will be created under reportdirectory